docs/volunteering/teams.md

```diff
@@ -0,0 +1,55 @@
+# Volunteer Teams
```

Comment:
Perhaps use a synonym for teams here if you're concerned about confusion versus competitor teams?
Maybe "organisation", "structure" or "groups"?

Reply:
The issue is that said groups of people are named "Team", so "Teams" is the logical plural. 

See also https://github.com/srobo/runbook/pull/76#issuecomment-578442154